ui_framework/doc_site/src/views/card/card_group.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+
+import {
+  KuiCardGroupContainer,
+  KuiCardGroup,
+  KuiCard,
+  KuiCardDescription,
+  KuiCardDescriptionTitle,
+  KuiCardDescriptionText,
+  KuiCardFooter
+} from '../../../../components';
+
+export default () => {
+  const cardStyle = {
+    width: '400px'
+  };
+
+  return (
+    <div>
+      <KuiCardGroupContainer>
+        <KuiCardGroup>
+          <KuiCard className="kuiCardGroup__card" style={cardStyle}>
```

Comment:
I really like this idea! But I'd like to explore it in a separate PR since some X-Pack code depends upon these styles. So let's leave it as-is for now.